index.js

```diff
@@ -9,6 +9,7 @@ Program
     .command('changelog', 'Echos a plugin zip file\'s changelog.')
     .command('compatibility', 'Updates the minimum Shopware version compatibility of all binaries of a plugin.')
     .command('description', 'Updates the plugin description of a supported locale.')
+    .command('dump-plugin', 'Outputs all information for a specified plugin.')
```

Comment:
This should probably include that it's dumped as JSON.

Reply:
I'm not sure if that option existed when I first reviewed. If it did, I must have missed that and my comment above is obsolete 😉

Reply:
It did ;)